dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -465,4 +465,36 @@ public ArrayList getOrganizations(@ApiParam(hidden = true) @Auth User authUser)
         }
         return null;
     }
+    @GET
+    @Timed
+    @UnitOfWork
+    @Path("/{userId}/bitbucketUser")
+    @ApiOperation(value = "Test bitbucket", notes = "NO authentication", response = String.class)
```

Comment:
Could you explain this more, is this a resource used purely for testing what the bitbucket user's identity is or would it be useful for other purposes?